sui/src/gateway_config.rs

```diff
@@ -117,7 +119,7 @@ impl Default for GatewayConfig {
             authorities: vec![],
             send_timeout: Duration::from_micros(4000000),
             recv_timeout: Duration::from_micros(4000000),
-            buffer_size: transport::DEFAULT_MAX_DATAGRAM_SIZE,
+            buffer_size: 650000,
```

Comment:
I probably should have left a comment, this is effectively unused as it was a configuration parameter specifically for the old networking stack. Instead of unthreading the whole config in this PR I just put in the old value of the const. After this PR it should be unused and I do plan on stripping it in a later PR.